app/bundles/FormBundle/EventListener/PageSubscriber.php

```diff
@@ -70,8 +93,8 @@ public function onPageBuild(PageBuilderEvent $event)
         }
 
         if ($event->tokensRequested($this->formRegex)) {
-            $tokenHelper = new BuilderTokenHelper($this->factory, 'form');
-            $event->addTokensFromHelper($tokenHelper, $this->formRegex, 'name', 'id');
+            $tokenHelper = new BuilderTokenHelper($this->mauticFactory, 'form');
```

Comment:
Keeping the factory here until the BuilderTokenHelper ticket gets merged, just noting so we don't forget.